### ISSUE TITLE
Turn off BluetoothLeScanner logs

### ISSUE THREAD

**Issue by SoCmodder** [AltBeacon/spec#10]
Not sure if I'm the only one with this issue, but it would be awesome if I could turn off the onClientRegistered() and onScanResult() logs. Or just all logging associated with the library. It makes debugging other features nearly impossible with the logs constantly filling up.

**Comment by yepher** [comment]
Not sure what platform your debugging on but if you are on a mac and need a quick solution to filtering Android logs try LogRabbit in Mac App Store.

Hope that helps until logging is fixed.

**Comment by mbeacons** [comment]
I'm sorry but not sure what this is? I didn't raise this?

**Comment by borzh** [comment]
Yes, please, this is very annoying

**Comment by ryot** [comment]
This issue has been reported to Google and is reportedly fixed as of September by their Android developers. Perhaps OS version 7.1.2 includes the fix? https://code.google.com/p/android/issues/detail?id=203965&thanks=203965&ts=1458051356

**Comment by mdxs** [comment]
I don't think this issue (still) belong in the https://github.com/AltBeacon/spec specification area. Could it be moved into the appropriate software oriented project if still a concern?
